include/ibeo_lux/ibeo_lux_common.h

```diff
@@ -1,5 +1,12 @@
-#ifndef IBEO_LUX_COMMON_H
-#define IBEO_LUX_COMMON_H
+/*
+ * Unpublished Copyright (c) 2009-2019 AutonomouStuff, LLC, All Rights Reserved.
+ *
+ * This file is part of the ibeo_lux ROS 1.0 driver which is released under the MIT license.
+ * See file LICENSE included with this software or go to https://opensource.org/licenses/MIT for full license details.
+ */
+
+#ifndef IBEO_LUX_IBEO_LUX_COMMON_H
```

Comment:
This looks weird, are you including the directory in there too?  I'm surprised that passed lint, it usually particular for me about the include guards being just the filename.

Reply:
If your include folder has the name of the package as a sub-folder (which it should), then `roslint` (cpplint) wants the folder name included too.